<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInterface</class>
 <widget class="QWidget" name="GameInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>864</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>双人五子棋</string>
  </property>
  <widget class="QLabel" name="gameArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>535</width>
     <height>535</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/picture/picture/background.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="regretRequesst_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>670</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitRequest_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>750</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>申请退出</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>780</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gamePrepare_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>780</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>游戏准备</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatEdit_textEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>610</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="announcement">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>公告:无</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatWindow_textEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveGame">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>保存游戏</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGame_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>760</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>陈代明硬笔体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>载入存档</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
